lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -730,7 +737,7 @@ def _get_node_information(self, node: Node) -> Dict[str, str]:
                     "security_profile"
                 ] = security_profile.security_profile.value
                 if security_profile.security_profile == SecurityProfileType.CVM:
-                    information["encrypt_disk"] = security_profile.encrypt_disk
+                    information["security_profile"] = security_profile.encrypt_disk
```

Comment:
Does it needs to know where the security_profile from? The log also have those information, why does it need to add into information?

Reply:
lisa customer wants to get these info in junit report, any suggestions?

Reply:
How about we add an extra switch like `collect_settings` on platform. Once it's enabled, add all features's settings into information.

Reply:
we already use capture_vm_information to control this logic, though default value for it is True.

Reply:
We should had a better name. For this one, it can be `capture_features_settings`. Once it's enabled, save all feature settings into the information of test results.